readme.md

```diff
@@ -74,10 +74,11 @@ To tail the logs from the Docker containers in realtime, run:
 docker-compose logs -f
 ```
 
-Local Setup
------------
-Don't want to set up a whole Vagrant VM or use Docker to try out Wagtail? No problem. You'll probably want to start
-with a fresh virtualenv.
+Traditional (Local) Setup
+-----------------------
+
+Want to run the bakerydemo locally without setting up Vagrant or Docker? You'll probably want to start
+with a fresh [virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/).
```

Comment:
Under 'Setup with Vagrant' we have

```
### Dependencies
* [VirtualBox](https://www.virtualbox.org/)
* [Vagrant 1.5+](http://www.vagrantup.com)
```

I'd suggest, for consistency, that we follow the same pattern. e.g
```
```
### Dependencies
* [Virtualenv](https://virtualenv.pypa.io/en/stable/installation/)
* [VirtualenvWrapper](http://virtualenvwrapper.readthedocs.io/en/latest/install.html)
```